doc/programming_paradigm.rst

```diff
@@ -119,10 +124,13 @@ As discussed later, each HOF's *operator* is an :term:`algorithm` registered wit
 | :ref:`Unfold <hof_operators:Unfolds>`                | :math:`p: D \rightarrow \mbox{Boolean}` | .. math::                                        | :math:`\dim(c) > \dim(d)`  | :math:`|c| > |d|`    |
 |                                                      |                                         |    :no-wrap:                                     |                            |                      |
 |                                                      |                                         |                                                  |                            |                      |
-|                                                      +-----------------------------------------+    \(                                            |                            |                      |
-|                                                      | :math:`q: D \rightarrow D \times C`     |    \underbrace{(d_{i_1\dots i_m})}_d \rightarrow |                            |                      |
+|                                                      | .. admonition:: Chris Green             |    \(                                            |                            |                      |
+|                                                      |    :class: admonition-chg               |    \underbrace{(d_{i_1\dots i_m})}_d \rightarrow |                            |                      |
 |                                                      |                                         |    \underbrace{(c_{i_1\dots i_n})}_c             |                            |                      |
-|                                                      |                                         |    \)                                            |                            |                      |
+|                                                      |    I don't understand this unfold form. |    \)                                            |                            |                      |
```

Comment:
It will be discussed in the section on unfolds.

Reply:
If you mean the one under 2.2.4, I'm not seeing it: should there be a TODO note there for now?